### ISSUE TITLE
Digits is shutdown right ? How this will work ?

### ISSUE THREAD

**Issue by myinnos** [Fenscode/twitter_digits_auth#1]
I think so better to develop firebase authentication directly instead of migrating from digits(new comers)

**Comment by ArslanKathia** [comment]
yeah you are right i am also asking this question because digits based on fabric and fabric now under in google.. Now digits is shutdown and go to digits console its also shown to you migrate your project to firebase phone authenication.
Thanks!
